app/javascript/profileDropdown/flagButton.js

```diff
@@ -58,10 +34,49 @@ export function initFlag() {
             flagButton.innerHTML = `Flag ${profileUserName}`;
           }
         })
-        .catch((e) => window.alert(`Something went wrong: ${e}`));
+        .catch((e) => {
+          Honeybadger.notify(
+            isUserFlagged ? 'Unable to unflag user' : 'Unable to flag user',
+            userData.profileUserID,
```

Comment:
I suspect this is an error (that name doesn't make any sense in this context) from copying directly from a suggestion.